core/actionProxy/Dockerfile

```diff
@@ -2,11 +2,8 @@
 FROM python:3.6.1-alpine
 
 # Upgrade and install basic Python dependencies.
-RUN apk add --no-cache bash \
- && apk add --no-cache --virtual .build-deps \
-        bzip2-dev \
-        gcc \
-        libc-dev \
+RUN apk add --no-cache bash perl jq zip git \
+  && apk add --no-cache --virtual .build-deps bzip2-dev gcc libc-dev \
```

Comment:
git?

Reply:
I thought it could be useful when I was thinking on usage of bash out of the box
For example having git useful if you get a Github webhook 
Or just want to git clone a repo to do a builds for CI